app/controllers/income/UpdateIncomeNextYearController.scala

```diff
@@ -40,7 +46,7 @@ trait UpdateIncomeNextYearController extends TaiBaseController
   with FeatureTogglesConfig
   with Auditable {
 
-  val updateNextYearsIncomeService: UpdateNextYearsIncomeService
+  def updateNextYearsIncomeService: UpdateNextYearsIncomeService
```

Comment:
Why has this changed? Whats the rational for it? The override in the object is still a val

Reply:
I'm not sure there was one, I can remove this since its unnecessary